app/manage/permissions/index.js

```diff
@@ -1,53 +1,22 @@
-const jwt = require('jsonwebtoken')
-const db = require('../db')
-const hydra = require('../lib/hydra')
+const db = require('../../db')
+const hydra = require('../../lib/hydra')
+const { mergeAll } = require('ramda')
 
-/**
- * Returns true if a jwt is still valid
- *
- * @param {jwt} decoded the decoded jwt token
- */
-function assertAlive (decoded) {
-  const now = Date.now().valueOf() / 1000
-  if (typeof decoded.exp !== 'undefined' && decoded.exp < now) {
-    return false
-  }
-  if (typeof decoded.nbf !== 'undefined' && decoded.nbf > now) {
-    return false
-  }
-  return true
+const metaPermissions = {
+  'public': () => true
 }
 
-/**
- * Attaches the user from the jwt to the session
- */
-function attachUser () {
-  return function (req, res, next) {
-    if (req.session) {
-      if (req.session.idToken) {
-        // We have an id_token, let's check if it's still valid
-        const decoded = jwt.decode(req.session.idToken)
-        if (assertAlive(decoded)) {
-          req.session.user_id = decoded.sub
-          req.session.user = decoded.preferred_username
-          req.session.user_picture = decoded.picture
-          return next()
-        } else {
-          // no longer alive, let's flush the session
-          req.session.destroy(function (err) {
-            if (err) next(err)
-            return next()
-          })
-        }
-      }
-    }
-    next()
-  }
+const teamPermissions = {
+  'teams:create': require('./create-team.js'),
```

Comment:
I think I understand why this is plural rather than singular, but it's a little weird. I'd be tempted to make it `team:create` just to keep it consistent.